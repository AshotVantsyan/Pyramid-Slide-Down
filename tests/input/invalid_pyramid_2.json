[[3], [7, 4], [8, 5, 9, 3]]