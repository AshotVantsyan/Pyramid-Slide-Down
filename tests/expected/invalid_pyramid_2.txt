Selected asset(s): tests/input/invalid_pyramid_2.json
Invalid pyramid. Expected 3 elements at 3 row, got 4